protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -2268,6 +2326,66 @@ func extractFieldBehaviorFromFieldDescriptor(fd *descriptorpb.FieldDescriptorPro
 	return opts, nil
 }
 
+func getFieldVisibilityOption(fd *descriptor.Field) (*visibility.VisibilityRule, error) {
```

Comment:
I think we can make this signature simpler by just returning a visibility rule that may or may not be nil, then we can skip the error check that we're not doing anything with anyway. What do you think?